proto/cosmos/sanction/v1beta1/events.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package cosmos.sanction.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/sanction";
+
+// EventAddressSanctioned is an event emitted when an address is sanctioned.
+message EventAddressSanctioned {
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// EventAddressUnsanctioned is an event emitted when an address is unsanctioned.
+message EventAddressUnsanctioned {
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// EventTempAddressSanctioned is an event emitted when an address is temporarily sanctioned.
+message EventTempAddressSanctioned {
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// EventTempAddressUnsanctioned is an event emitted when an address is temporarily unsanctioned.
+message EventTempAddressUnsanctioned {
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// EventParamsUpdated is an event emitted when the sanction module params are updated.
+message EventParamsUpdated {}
```

Comment:
Was this planned to have some information in it?  Just emitted to signal a param has been updated?

Reply:
Just emitted to signal the params have been updated. The only thing to possibly put in there are the params, but that doesn't feel like something that would be in an event. The params can only be updated via gov prop, and I'm not even sure how such events end up in a block since they're not part of a tx.